src/java.base/macosx/native/libjli/java_md_macosx.m

```diff
@@ -87,10 +87,6 @@
  * The vm selection options are not passed to the running
  * virtual machine; they must be screened out by the launcher.
```

Comment:
The flowchart below seems to be out-of-date as I'm pretty sure no data-model selection is performed any more (no -d64).